examples/app/server_with_auto_scaler/app.py

```diff
@@ -26,7 +26,12 @@ def __init__(self, *args, **kwargs):
         )
 
     def setup(self):
-        self._device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+        if torch.cuda.is_available():
```

Comment:
As this is used several times, can we wrap it into a function?

Reply:
Hi, Where should I keep the function?

Reply:
seems I was wrong as each example shall be stand-alone...